_includes/section.html

```diff
@@ -1,47 +1,45 @@
 {%- comment -%}
-  Include vars:
-	title
-  subtitle
-	photo
-	photo-alt
-	auto-header
-	content
+Include vars:
+title
+subtitle
+photo
+photo-alt
+auto-header
+content
 {%- endcomment -%}
 
 {%- assign _title = include.title | default: 'Untitled' -%}
 {%- assign _slug = include.title | strip_html | slugify | default: 'untitled' -%}
 
 {%- if include.photo and include.photo-alt -%}
-	{%- capture _photo -%}
-		<img src="{{- include.photo | relative_url -}}" alt="{{- include.photo-alt -}}" class="back">
-	{%- endcapture -%}
-	{%- assign _class = 'shade-one cover dark' -%}
+{%- capture _photo -%}
+<img src="{{- include.photo | relative_url -}}" alt="{{- include.photo-alt -}}" class="back">
+{%- endcapture -%}
+{%- assign _class = 'shade-one cover dark' -%}
 {%- else -%}
-	{%- assign _photo = '' -%}
-	{%- capture _class -%}shade-{%- cycle 'two', 'three', 'four', 'three' -%}{%- endcapture -%}
+{%- assign _photo = '' -%}
```

Comment:
but now you removed all the indentation in some places and that wasn't the point :sweat_smile: